src/platforms/java/common/gradle.mdx

```diff
@@ -33,13 +33,13 @@ buildscript {
 }
 
 plugins {
-    id("io.sentry.android.gradle") version "{{@inject packages.version('sentry.java.android.gradle-plugin', '3.0.0') }}"
+    id("io.sentry.jvm.gradle") version "{{@inject packages.version('sentry.java.android.gradle-plugin', '3.9.0') }}"
 }
 ```
 
 <Note>
 
-Although the plugin id contains `android`, it can be used for non-android projects. However, for non-android projects, the plugin only collects external dependencies.
+Although the plugin repository name contains `android`, it can be used for non-android projects. However, for non-android projects, the plugin only collects external dependencies and uploads <PlatformLink to="/source-context/">Source Context</PlatformLink>.
```

Comment:
```suggestion
Although the plugin repository name contains `android`, it can be used for non-Android projects.

For non-Android projects, the plugin only collects external dependencies and uploads <PlatformLink to="/source-context/">Source Context</PlatformLink>.
```